http/swagger.yml

```diff
@@ -11047,6 +11189,55 @@ components:
     NotificationEndpointType:
       type: string
       enum: ['slack', 'pagerduty', 'http']
+    DBRP:
+      properties:
+        orgID:
```

Comment:
We will need an `id` property added to the Swagger. This will necessary so the Get, Update and Delete operations can function correctly with an addressable ID:

```yaml
        id:
          readOnly: true
          type: string
```

That will require we update the `DBRPMapping` struct to include an `ID` property:

https://github.com/influxdata/influxdb/blob/e4539ad04d5fb6f7bc3b0e2516ebc5812036ee49/dbrp_mapping.go#L26-L36

We will also need to amend the `DBRPMappingService` to be able to Get, Update and Delete a mapping by `ID`:

```go
// DBRPMappingService provides a mapping of cluster, database and retention policy to an organization ID and bucket ID.
type DBRPMappingService interface {
	FindMappingByID(ctx context.Context, id ID) (*DBRPMapping, error)
	UpdateMapping(ctx context.Context, id ID, update DBRPMappingUpdate) (*DBRPMapping, error)
	DeleteMapping(ctx context.Context, id ID) error
}
```